tests/cycpp_tests.py

```diff
@@ -581,24 +843,25 @@ def test_nuclide_uitype():
     impl = f.impl()
     exp_impl = ('  return ""\n'
                 '    "<interleave>\\n"\n'
-                '    "<element name=\\"x\\">\\n"\n'
-                '    "    <data type=\\"string\\" />\\n"\n'
-                '    "</element>\\n"\n'
-                '    "</interleave>\\n"\n    ;\n')
+                '    "    <element name=\\"x\\">\\n"\n'
+                '    "        <data type=\\"string\\"/>\\n"\n'
+                '    "    </element>\\n"\n'
+                '    "</interleave>\\n";\n')
     yield assert_equal, exp_impl, impl
 
     # test infiletodb updates
     f = InfileToDbFilter(m)
     f.given_classname = 'MyFactory'
     impl = f.impl()
-    exp_impl = ('  tree = tree->SubTree("config/*");\n'
-                '  cyclus::InfileTree* sub;\n'
-                '  int i;\n'
-                '  int n;\n'
-                '  x = pyne::nucname::id(cyclus::Query<std::string>(tree, "x"));\n'
-                '  di.NewDatum("Info")\n'
-                '  ->AddVal("x", x)\n'
-                '  ->Record();\n')
+    exp_impl = (
+        '  cyclus::InfileTree* sub = tree->SubTree("config/*");\n'
+        '  int i;\n'
+        '  int n;\n'
+        '{  int x_val = pyne::nucname::id(cyclus::Query<std::string>(sub, "x"));\n'
+        'x = x_val;}  di.NewDatum("Info")\n'
```

Comment:
This does not seem properly formatted.  I was seeing the same thing in the output of the sink.h